src/vmware/azext_vmware/_help.py

```diff
@@ -175,14 +195,33 @@
 
 helps['vmware datastore create'] = """
     type: command
-    short-summary: Create a datastore in a private cloud cluster.
+    short-summary: Please use "net-app-volume create" or "disk-pool-volume create" instead.
+"""
+
+helps['vmware datastore net-app-volume'] = """

```

Comment:
Would it be possible to rename `net-app-volume`  to `netapp-volume`?

Reply:
Renamed in commit e56d2e1d227100c126d16c302598b960ce1cf938.